packages/p2p/src/stores/order-store.js

```diff
@@ -136,8 +136,6 @@ export default class OrderStore {
                         response?.error.code === api_error_codes.INVALID_VERIFICATION_TOKEN ||
                         response?.error.code === api_error_codes.EXCESSIVE_VERIFICATION_REQUESTS
                     ) {
-                        // TODO: remove this once refactoring of EmailLinkVerifiedModal is done
-                        this.root_store.general_store.hideModal();
```

Comment:
@farrah-deriv can I remove this, since this comment was previously added in your PR? There's no issues atm when this statement is deleted

Reply:
sure @adrienne-deriv